.github/workflows/docs.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     container: ghcr.io/acts-project/ubuntu2004:v29
     env:
-        DOXYGEN_WARN_AS_ERROR: YES
+        DOXYGEN_WARN_AS_ERROR: FAIL_ON_WARNINGS
```

Comment:
shouldn't that disappear with https://github.com/acts-project/acts/pull/1497/files ? or is a merge from master missing?

Reply:
Yes it should, as soon as this branch is updated it should be gone I believe.